Fix Deploy Github Actions

```
yarn install v1.22.10
[1/4] Resolving packages...
[2/4] Fetching packages...
warning Pattern ["ajv-latest@npm:ajv@^8.6.0"] is trying to unpack in the same destination "/home/runner/.cache/yarn/v6/npm-ajv-latest-8.6.0-60cc45d9c46a477d80d92c48076d972c342e5720-integrity/node_modules/ajv-latest" as pattern ["ajv@^8.0.1"]. This could result in non-deterministic behavior, skipping.
[1/4] Resolving packages...
[2/4] Fetching packages...
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
warning jest > @jest/core > jest-haste-map > sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
warning jest > @jest/core > jest-haste-map > sane > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning jest > @jest/core > jest-haste-map > sane > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
[2/4] Fetching packages...
error An unexpected error occurred: "ENOENT: no such file or directory, scandir '/home/runner/.cache/yarn/v6/npm-kleur-3.0.3-a79c9ecc86ee1ce3fa6206d1216c501f147fc07e-integrity/node_modules/kleur'".
info If you think this is a bug, please open a bug report with the information provided in "/home/runner/work/graasp-compose/graasp-compose/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/deployProd.yml
+++ b/.github/workflows/deployProd.yml
@@ -1,11 +1,13 @@
-name: deploy prod build to aws
+name: deploy production build to aws
 
 on:
   push:
-    branches:
-      - master
+    # Sequence of patterns matched against refs/tags
+    tags:
+      # Push events to matching v*, i.e. v1.0, v20.15.10
+      - 'v*'
 jobs:
-  deploy-dev:
+  deploy-prod:
     runs-on: ubuntu-latest
     env:
       # default build directory
@@ -25,6 +27,10 @@ jobs:
       - name: yarn set up
         uses: actions/setup-node@v2
 
+      # as long as github dependencies / dependencies with prepare scripts are needed
+      # this script might encounter trouble when installing
+      # https://github.com/yarnpkg/yarn/issues/6312
+      - run: rm yarn.lock
       - run: yarn install --network-concurrency 1
       - run: yarn build
 

